monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.sla

```diff
@@ -21680,9 +21680,9 @@
             <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="14" CH="A través de una sofisticada criptografía de "/>
             <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="14" FEATURES="inherit underline" CH="firmas circulares"/>
             <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="14" CH=", "/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="14" FEATURES="inherit underline" CH="firmas confidenciales de anillo"/>
+            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="14" FEATURES="inherit underline" CH="firmas confidenciales de círculo"/>
             <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="14" CH=" y "/>
-            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="14" FEATURES="inherit underline" CH="stealth addresses"/>
+            <ITEXT FONT="Adobe Caslon Pro Bold Italic" FONTSIZE="14" FEATURES="inherit underline" CH="direcciones de sigilo"/>
```

Comment:
What I noticed is that it seems we are trying not to use the feminine words, in any case, not even if a masculine word ends in a plural "feminine" way and can be used or "replace" the singular masculine, saying the exact same words. This won't hurt any type of preference in the gender terminology, because is normal to write like that. 

Example (btw, I have never been ok with this translations but we are going to work it out and see how it goes): **Address = Dirección**, and "dirección" is feminine, there is no way on making addresses masculine.
- Singular
"Stealth Address" = "Direccion de sigilo"
- Plural
"Stealth Addresses" = "Direcciones sigilosas (feminine) or Direcciones de sigilo(**"masculine"**)"

*Both can be used, but we can't get into a discussion or "fight" because one expresses itself in plural and the other one in singular plural.

A person can either be "sigilosa" or "sigiloso", and if we say "sigiloso" we will always be marking a "masculine" PERSON gender word. If we say a person is "sigilosa"(una persona es sigilosa) is not referring a feminine gender terminology person (sex, he or she), only as plural of "sigilo". When the word, in this case "address = dirección" (feminine) is next to a singular word "sigilo" or plural "sigilos" (masculine noun), is not right to say "dirección de sigilos", you could either say "direcciones sigilosas" or like the suggested used words "direcciones de sigilo".

I leave this comment because it seems you are avoiding grammatical structures that are agreed in the spanish vocabulary and can also be used without affecting or giving gender preferences.

Reply:
@BlackLotus64 ping

Reply:
Waiting for the results of the poll in Telegram :)

Reply:
The poll ended and the results are: _dirección secreta_. Updating.

Reply:
Great community support! :)